app/assets/javascripts/script.js

```diff
@@ -0,0 +1,7 @@
+$(function() {
+  $('#new_inquiry').on('ajax:complete', function(e, data, status, xhr){
+    $('#inquiry_message').val('Thank you, your inquiry was sent.');
+
+  $('#new_conversation').on('ajax:complete', function(e, data, status, xhr){
```

Comment:
'xhr' is defined but never used.<br>'status' is defined but never used.<br>'data' is defined but never used.<br>'e' is defined but never used.